assets/src/application/ui/display/icons/collection/Sort.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { IconProps, IconName } from '../types';
+import Icon from '../Icon';
+
+const Sort = React.forwardRef<SVGSVGElement, Partial<IconProps>>((props, ref) => (
+	<Icon {...props} name={IconName.SORT} ref={ref} />
+));
+
+Sort.displayName = 'Sort';
+
+export default Sort;
```

Comment:
on the meeting you said smth about generating react components from SVG, would be handy and less time-consuming to have a build script similar to the one described in this article: https://www.robinwieruch.de/react-svg-icon-components